hack/run-e2e-kind.sh

```diff
@@ -376,6 +376,50 @@ function setup-mcad-env {
 
 }
 
+function extend-resources {
+    # Patch nodes to provide GPUs resources without physical GPUs.
+    # This is intended to allow testing of GPU specific features such as histograms.
+
+    # Start communication with cluster
+    echo -n "Starting proxy "
+
+    kubectl proxy > /dev/null 2>&1 &
+    PROXY_PID=$!
+
+    echo "(pid=${PROXY_PID})..."
+
+    # Variables
+    RESOURCE_NAME="nvidia.com~1gpu"
```

Comment:
since this is inside a function you can consider using local variables, and perhaps use lower case names. Just a suggestion.

Reply:
Done